godot-macros/src/lib.rs

```diff
@@ -197,12 +197,18 @@ mod util;
 ///     #[export]
 ///     my_field: i64,
 /// }
+///
+/// #[godot_api]
+/// impl MyStruct {}
```

Comment:
it works apparently lol, this doctest failed because it was originally lacking this impl.